src/figuresClasses.ts

```diff
@@ -1,19 +1,87 @@
-export interface Figure {
 
+// function triangle(a: number, b: number, c: number): boolean {
+//   if (a + b < c || a + c < b || c + b < a) {
+//     return false;
+//   }
+
+//   if (a < 0 || b < 0 || c < 0) {
+//     return false;
+//   }
+
+//   return true;
+// }
+
+enum Shape {
+  triangle,
+  circle,
+  rectangle,
+}
+
+enum Color {
+  red,
+  green,
+  blue,
+}
+
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  a: number,
+  b: number,
+  c: number,
+  getArea(): number,
 }
 
 export class Triangle {
+  // eslint-disable-next-line no-useless-constructor
+  constructor(
+    public shape: string,
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {}
```

Comment:
your constructor shouldn't be empty, read documentation, please, 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes
f.e
![image](https://user-images.githubusercontent.com/59209337/215262563-d5dac1d3-a2f1-4e7d-82c9-c7b231a104b7.png)